GherkinSyncTool.Synchronizers.AllureTestOps/AllureTestOpsSynchronizer.cs

```diff
@@ -79,35 +82,56 @@ public void Sync(List<IFeatureFile> featureFiles)
                         {
                             try
                             {
-                                _allureClientWrapper.UpdateTestCase(allureTestCase, caseRequest);    
+                                _allureClientWrapper.UpdateTestCase(allureTestCase, caseRequestExtended);
                             }
                             catch (AllureException e)
                             {
-                                Log.Error(e, $"The test case has not been updated: {caseRequest.Name}");
+                                Log.Error(e, $"The test case has not been updated: {caseRequestExtended.CreateTestCaseRequest.Name}");
                                 _context.IsRunSuccessful = false;
                             }
                         }
                     }
                 }
             }
-
+            //TODO:
+            DeleteNotExistingScenarios(allureTestCases, featureFilesTagIds);
             Log.Debug(@$"Synchronization with Allure TestOps finished in: {stopwatch.Elapsed:mm\:ss\.fff}");
         }
 
-        private TestCase CreateTestCase(CreateTestCaseRequest caseRequest)
+        private void DeleteNotExistingScenarios(List<TestCaseContent> allureTestCases, List<ulong> featureFilesTagIds)
+        {
+            //TODO:
```

Comment:
It seems to me that it's better to throw `NotImplementedException` here with appropriate message

Reply:
Done